test/OptimismCrosschainTest.t.sol

```diff
@@ -30,39 +25,35 @@ contract OptimismCrosschainPayload is CrosschainPayload {
 
 contract OptimismCrosschainTest is CrosschainTestBase {
 
+    using DomainHelpers         for *;
+    using OptimismBridgeTesting for *;
+
     function deployCrosschainPayload(IPayload targetPayload, address bridgeReceiver)
         public override returns (IPayload)
     {
         return IPayload(new OptimismCrosschainPayload(targetPayload, bridgeReceiver));
     }
 
     function setUp() public {
-        hostDomain = new Domain(getChain('mainnet'));
-        bridgedDomain = new OptimismDomain(getChain('optimism'), hostDomain);
+        bridge = OptimismBridgeTesting.createNativeBridge(getChain('mainnet').createFork(), getChain('optimism').createFork());
 
```

Comment:
Lets apply this formatting change to all `createNativeBridge` lines just to keep lines shorter

Reply:
```suggestion
        bridge = OptimismBridgeTesting.createNativeBridge(
            getChain('mainnet').createFork(),
            getChain('optimism').createFork()
        );
```

Reply:
done